django_redis/client/default.py

```diff
@@ -15,16 +19,15 @@
 
 _main_exceptions = (TimeoutError, ResponseError, ConnectionError, socket.timeout)
 
-
 special_re = re.compile("([*?[])")
 
 
-def glob_escape(s):
+def glob_escape(s) -> Union[bytes, str]:
```

Comment:
The regex is `str`, not `bytes`. Did you find it necessary to include `bytes`? If this is intended to support both, it is buggy and will produce a `BytesWarning`.

Reply:
Yeah, I fixed it but did not push the modifications yet

Reply:
Sorry for that